benchmarks/datasets/join/create_index/pg_search.sql

```diff
@@ -0,0 +1,75 @@
+CREATE INDEX pages_index ON pages
+USING bm25 ("id",
+    "content",
+    "title",
+    "parents",
+    "fileId",
+    "sizeInBytes",
+    "createdAt"
+)
+WITH (
+    key_field = 'id',
+    text_fields = '{
+    	"fileId": {
+    		"tokenizer": { "type": "keyword" }, "fast": true
+    	},
+        "content": {
+            "tokenizer": { "type": "icu" }
+        },
+        "title": {
+            "tokenizer": { "type": "icu" }
+        },
+        "parents": {
+            "tokenizer": { "type": "icu" }
+        }
```

Comment:
IMHO, let's mark them all as `fast` and then choose the execution method via GUCs.
```suggestion
        "content": {
            "tokenizer": { "type": "icu" }, "fast": "true"
        },
        "title": {
            "tokenizer": { "type": "icu" }, "fast": "true"
        },
        "parents": {
            "tokenizer": { "type": "icu" }, "fast": "true"
        }
```